pkg/generated/clientset/core/internalclientset/typed/core/v1alpha1/csivolume.go

```diff
@@ -132,6 +133,22 @@ func (c *cSIVolumes) Update(cSIVolume *v1alpha1.CSIVolume) (result *v1alpha1.CSI
 	return
 }
 
+// UpdateStatus was generated because the type contains a Status member.
+// Add a +genclient:noStatus comment above the type to avoid generating UpdateStatus().
+
+func (c *cSIVolumes) UpdateStatus(cSIVolume *v1alpha1.CSIVolume) (result *v1alpha1.CSIVolume, err error) {
```

Comment:
sorry for late in they cycle.. but, `cSIVolumes` doesn't look good.. it should be `csiVolumes` or `CSIVolumes`
acronyms need to be of same type letter

Reply:
This is generated code.